ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -567,6 +581,7 @@ private[state] object Conversions {
           buildStatus(
             Code.ABORTED,
             s"Inconsistent: Missing input state for key ${rejection.getKey.toString}",
+            Map("key" -> rejection.getKey.toString),
```

Comment:
This will produce an output with newlines. Given that the goal of these pieces of metadata is not to print them out on the console, I'd suggest using [`shortDebugString`](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/TextFormat.Printer.html#shortDebugString-com.google.protobuf.MessageOrBuilder-) instead, as that doesn't produce newlines and is still human readable.

Reply:
Sorry, this is something that has been merged in the previous PR. We decided to leave this one, as it's possible to parse it with: `TextFormat.parse`, and this is what is demonstrated by a test. Are you ok with that?